libraries/src/CMS/Application/ApiApplication.php

```diff
@@ -55,6 +63,11 @@ public function __construct(JInputJson $input = null, Registry $config = null, W
 		// Execute the parent constructor
 		parent::__construct($input, $config, $client, $container);
 
+		$this->triggerEvent('onGetApiFormats', [$this->formatMapper]);
```

Comment:
Let's try to not do this in the constructor.  Seems we're already having issues with events triggering too early with `onBeforeExecute` having been added.  This should in theory be safe to handle with `onAfterInitialise`.